py-deploy/tldeploy/cli.py

```diff
@@ -81,6 +81,12 @@ def cli(ctx, version):
     show_default=True,
 )
 @currency_network_contract_name_option
+@click.option(
+    "--expiration-time",
+    help="Expiration time of the currency network after which it will be frozen",
+    default=4102444800,  # 01/01/2100
+    show_default=True,
+)
 @jsonrpc_option
```

Comment:
Please add the functionality so the user can specify a date and time combination in addition to the expiration-time integer.
(look at the auction on how it's done there).

Reply:
Then I will either have no default value and break everything that uses this cli command currently, or I will have no check for user inputting both a date and a timestamp (wrong input).

I chose to copy what is done in the auction and have no default value.